src/wp-includes/media.php

```diff
@@ -1593,6 +1606,42 @@ function wp_image_add_srcset_and_sizes( $image, $image_meta, $attachment_id ) {
 	return $image;
 }
 
+/**
+ * Filters 'img' and 'iframe' elements in post content to add 'loading' attributes.
+ *
+ * @since 5.4.0
+ *
+ * @param string $content The raw post content to be filtered.
+ * @return string Converted content with 'loading' attributes added to images.
+ */
+function wp_lazy_load_content_media( $content ) {
+	$tags = array( 'img', 'iframe' );
```

Comment:
Might as well move this definition down below the applied filter, as it will only be used if the function is not short-circuited.

Reply:
I am thinking that too. I also think we should have a filter on these tags, as a user may want to remove/ add filter tags to this list.

Reply:
I think it is premature to add a filter for this. For adding, as only `img` and `video` support the attribute so it doesn't make sense to remove. For removing, perhaps better to wait for there to actually be a clear use case before adding something that perhaps would never be used.

Reply:
Agreed, let's not introduce more filters than needed. These two elements are the only supported ones today.

Reply:
I think there would be kick back if we don't allow developers to disable iframes for example. Iframes and images can be used for tracking.